docs/patterns/components/Modal/index.js

```diff
@@ -144,6 +146,54 @@ class Demo extends Component {
 }
           `}
         </Code>
+        <div className="Demo-props">
+          <h2>Props</h2>
+          <PropDocs
+            docs={{
+              children,
+              className,
+              variant: {
+                type: 'string',
+                description: 'The style of Modal to display.',
+                default: 'default'
+              },
+              heading: {
+                type: 'React.ReactElement<any> or object',
+                description:
+                  'Displayed in the heading at the top of the Modal. Optional to pass heading level.',
+                required: true
+              },
+              onClose: {
+                type: 'function',
+                description: 'Function called when the Modal is closed'
+              },
+              portal: {
+                type: 'any',
+                description: 'The parent element to place the Modal in.',
```

Comment:
```suggestion
                description: 'The parent element to render the Modal within.',
```

Reply:
Updated this one in Modal, Tooltip, and TooltipTabstop since I copied and pasted 😁